pkg/kube/roles.go

```diff
@@ -64,10 +64,16 @@ func GetEnvironmentRoles(jxClient versioned.Interface, ns string) (map[string]*v
 
 // UpdateUserRoles updates the EnvironmentRoleBinding values based on the given userRoles
 // userKind is "User" or "ServiceAccount"
-func GetUserRoles(jxClient versioned.Interface, ns string, userKind string, userName string) ([]string, error) {
+func GetUserRoles(kubeClient kubernetes.Interface, jxClient versioned.Interface, ns string, userKind string, userName string) ([]string, error) {
```

Comment:
I felt passing in the `kubeClient` was a better option than passing in the admin namespace